README.md

```diff
@@ -60,7 +60,14 @@ Once you have implemented a lower bound search for the start of the range, imple
 
 *How do you use binary search to find the lower bound of a number? How did you have to modify the binary search algorithm?*
 
+-When we use binary search to determine the lower bound, we return the idx of the smallest element equal to or larger than v instead of the idx of the element equal to v.
+We find this by searching in the upper half of x if x[mid]<v and by searching in the lower half if x[mid]>v. This is the same as in the ordinary binary search. In contrast to binary search, we continue to search in the lower half if x[mid]=v, to see if there is an element at a lower idx equal to v, and we do not stop searching until low = high.
+
 *Would anything be more difficult if the features covered ranges instead of single nucleotides (like real BED files)? What could go wrong, if anything?*
 
+- We would also have to comepare the end values for all the features with the upper value as one might start within the interval but end beyond the upper value. 
```

Comment:
It is even more complicated than that. Generally, we want to find all feature-intervals that overlap the query-interval. But feature intervals could start long before and end long after. Finding all intervals that start or stop inside the query will not be enough. The general problem is a lot harder than what we did here (which is why we restricted the project to single nucleotide features).